### PR TITLE
Add Image namespace for Add-Ons service and Pod Infra Container Image 

### PR DESCRIPTION
**Pull request by xiaoluhong**
Add Image namespace for Add-Ons service and Pod Infra Container Image 

issue:https://github.com/rancher/rancher/issues/9853

@yasker  Hi, I submitted a new PR.

### CODE REVIEW

**Comment by deniseschannon** [comment]
@hongxiaolu please edit the templates in folder 33 with the changes as that is the folder that we are testing to ship with the end of september release.

**Comment by yasker** [comment]
@deniseschannon So we no longer need to create a new folder for the change?

**Comment by deniseschannon** [comment]
For this release, correct, we haven't released folder 33 to the public so we should keep editing/updating that. @yasker

**Comment by xiaoluhong** [comment]
@deniseschannon Hi，it's Updated, please review. thanks。

File: infra-templates/k8s/33/rancher-compose.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .catalog:
   name: Kubernetes
-  version: v1.8.0-beta.1-rancher2-etcdv2
+  version: v1.8.0-beta.1-rancher4
```
Comment (deniseschannon) [inline infra-templates/k8s/33/rancher-compose.yml]: please make sure to add in the `etcdv2` - do not remove that.

**Comment by xiaoluhong** [comment]
@deniseschannon Hi，it's Updated, please review.

**Comment by deniseschannon** [comment]
LGTM from a catalog perspective. @yasker can you check for the changes?

**Comment by yasker** [comment]
LGTM. I will merge soon.

**Comment by xiaoluhong** [comment]
Thank you all.

**Pull request merged**